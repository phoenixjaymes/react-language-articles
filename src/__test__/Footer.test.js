import React from 'react';
import ReactDOM from 'react-dom';
import { create } from 'react-test-renderer';
import Footer from '../components/Footer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

describe('Footer', () => {
  it('matches the snapshot', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
