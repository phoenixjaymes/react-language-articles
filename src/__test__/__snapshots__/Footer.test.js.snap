// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `
<footer>
  <div
    className="inner-container"
  >
    <small>
      Â© 2019 PhoenixJaymes
    </small>
  </div>
</footer>
`;
